compiler/hash-lower/src/build/into.rs

```diff
@@ -193,16 +192,13 @@ impl<'tcx> BodyBuilder<'tcx> {
                     //  | dest = true  |----------------+-->| join |
                     //  +--------------+                    +------+
                     // ```
-                    FnCallTermKind::LogicalBinOp(op, _, _) => {
+                    FnCallTermKind::LogicalBinOp(op, lhs_term, rhs_term) => {
                         let (short_circuiting_block, mut else_block, join_block) = (
                             self.control_flow_graph.start_new_block(),
                             self.control_flow_graph.start_new_block(),
                             self.control_flow_graph.start_new_block(),
                         );
 
-                        let lhs_term = self.stores().args().get_at_index(*args, 1).value;
-                        let rhs_term = self.stores().args().get_at_index(*args, 2).value;
```

Comment:
Might be responsible for different IR gen?

Reply:
Not sure what you mean

Reply:
Im investigating why ir changed, its a self comment

Reply:
Yeah, this is actually a bug in lowering that I accidentally discovered, it should be that I need to get the `2` and `3` arguments.